WindowsInstaller/config_template.txt

```diff
@@ -81,108 +40,298 @@ allow_user_actions_while_paused = false-----------------------------------------
+-- Free Build or Sandbox mode
+-- You cannot win or lose custom made maps if this is set to true.
+-- You also don't have to worry about money.
+-- This setting does not apply to any of the campaign maps.
+--
+free_build_mode = false
 
-width = SCREEN_SIZE_WIDTH
-height = SCREEN_SIZE_HEIGHT
-fullscreen = SCREEN_FULLSCREEN
--------------------------------------------------------------------------------
+----------------------------------------------- OPTIONS MENU --------------------------------------
+--These settings can also be changed from within the game from the options menu
+---------------------------------------------------------------------------------------------------
+-- Sounds: By default enabled and set at level 0.5
+--
+play_sounds = true
+sound_volume = 0.5
+
+---------------------------------------------------------------------------------------------------
+-- Announcements: By default set at level 0.5
+--
+play_announcements = true
+announcement_volume = 0.5
+
+---------------------------------------------------------------------------------------------------
+-- Background music: By default enabled and set at level 0.5
+--
+play_music = true
+music_volume = 0.5
+
+---------------------------------------------------------------------------------------------------
+-- Edge scrolling: By default enabled (prevent_edge_scrolling = false).
+--
+prevent_edge_scrolling = false
+
+---------------------------------------------------------------------------------------------------
+-- Capture mouse: By default enabled (capture mouse = true).
+--
+capture_mouse = true
+
+---------------------------------------------------------------------------------------------------
+-- Adviser on/off: If you set this to true the adviser will no longer
+-- pop up.
+--
+adviser_disabled = false
+
+---------------------------------------------------------------------------------------------------
+-- Scrolling Momentum.
+-- Determines the amount of momentum when scrolling the map with the mouse.
+-- This should be a value between 0 and 1 where 0 is no momentum
+--
+scrolling_momentum = 0.8
+
+---------------------------------------------------------------------------------------------------
+-- Top menu clock is by default is always on
+-- setting to true will give you a twentyfour hours display
+-- change to false if you want AM / PM time displayed.
+--
+twentyfour_hour_clock = true
+
+---------------------------------------------------------------------------------------------------
+-- Automatically check for updates.
+-- If set to true, CorsixTH will automatically check for and alert you to newer
+-- versions on startup.
+--
+check_for_updates = true
+
+---------------------------------------------------------------------------------------------------
+-- Warmth Colors display settings.
+-- This specifies which display method is set for warmth colours by default.
+-- Possible values: 1 (Red), 2 (Blue Green Red) and 3 (Yellow Orange Red).
```

Comment:
Is this for the heat map?

Reply:
Yes

Reply:
Perhaps this needs a bit more clarification then. Originally I thought it was referring to colour-blindness or overall screen colour (think f.lux)